frontend/packages/kubevirt-plugin/src/components/vm-templates/table/RowActions.tsx

```diff
@@ -86,14 +89,25 @@ type RowActionsProps = {
   disableCreate?: boolean;
 };
 
-const RowActions: React.FC<RowActionsProps> = ({
+const RowActions: React.FC<RowActionsProps & ModalComponentProps> = ({
   template,
   sourceStatus,
   namespace,
   disableCreate,
+  close,
 }) => {
   const { t } = useTranslation();
+  const permissionsErrorMsg = t(
+    'kubevirt-plugin~You do not have permissions to create Virtual Machine. Contact your system administrator for more information.',
+  );
+  const modalTitle = t('kubevirt-plugin~Create Virtual Machine from template');
```

Comment:
can I suggest putting `permissionsErrorMsg` and `modalTitle` to be inside the `permissionsErrorModal` component?
than you could change also the names to something shorter like `error` or `errorMsg` and `title`